### PR TITLE
[#2537] Integrate user_actions with JMS

### LINKED ISSUE #2537: Wrap as a component the direct way of publishing user actions
We do not resign from the direct way of publishing user actions to the recommender.
1) This method should be packaged into a component,
2) An environment variable should be created that will allow defining how we want to publish user actions (direct, databus or both),
3) Possibility of connecting to any queue (databus).

### PR DESCRIPTION
**Pull request by wujuu**
closes #2537

### CODE REVIEW

File: config/stomp_publisher.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ default: &default
   host: <%= ENV["MP_STOMP_HOST"] || Rails.application.credentials.stomp.dig(:host) || "''" %>
   ssl-enabled: <%= ENV["MP_STOMP_SSL"] || false %>
   topic: <%= ENV["MP_STOMP_PUBLISHER_TOPIC"] || Rails.application.credentials.stomp.dig(:publisher, :topic) || "''" %>
+  recommender-topic: <%= ENV["MP_STOMP_PUBLISHER_RECOMMENDER_TOPIC"] || "recommender" %>
```
Comment (Michal-Kolomanski) [inline config/stomp_publisher.yml:8]: - I wonder if "recommender-topic" is the best name. I would rather use something like "user_action_topic" in that case.
- Add a description of this variable to the README

**Review by Michal-Kolomanski — CHANGES_REQUESTED** [review]
The [scope](https://github.com/cyfronet-fid/marketplace/issues/2537) of the ticket was also to create an env variable to control user_actions destination. This variable should support 3 values. Sending user_actions:
1)  directly to the recommender,
2)  sending to databus, 
3)  both ways at the same time.

For now, it should be set to the third option. 

Furthermore, the whole mechanism should be extensively tested.

**Review by jswk — COMMENTED** [review]
~~Other than the misleading docs the PR looks ok.~~

**Review by jswk — CHANGES_REQUESTED** [review]
Please also add the test for the added behaviour to `spec/services/jms/publisher_spec.rb`.

**Comment by wujuu** [comment]
> Please also add the test for the added behaviour to `spec/services/jms/publisher_spec.rb`.

I've added tests, but to the UserActionController, because the Jms::Publisher didn't actually change. (it was Jms::Publish helper class). Left a TODO (for another PR) to expand the tests, possibly a task for @Michal-Kolomanski

**Comment by jswk** [comment]
> > Please also add the test for the added behaviour to `spec/services/jms/publisher_spec.rb`.
> 
> I've added tests, but to the UserActionController, because the Jms::Publisher didn't actually change(rather Jms::Publish helper class, which we do not test as for now - I've felt that we should test the UserActionController, because it enapsulates more of the logic added). Left a TODO (for another PR) to expand the tests, possibly a task for @Michal-Kolomanski

Right, my bad. You're right that the Jms::Publish doesn't have tests.

**Review by jswk — APPROVED** [review]
Seems ok now.

**Review by Michal-Kolomanski — APPROVED** [review]

**Comment by Michal-Kolomanski** [comment]
We need to inform Wojtek about the variables we've added and what values they should have.

**Pull request merged**